server/http_status.go

```diff
@@ -85,9 +87,38 @@ func (s *Server) startHTTPServer() {
 	serverMux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	serverMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
+	var (
+		err            error
+		httpRouterPage bytes.Buffer
+		pathTemplate   string
+	)
+	httpRouterPage.WriteString("<html><head><title>TiDB Status and Metrics Report</title></head><body><h1>TiDB Status and Metrics Report</h1><table>")
+	err = router.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
+		pathTemplate, err = route.GetPathTemplate()
+		if err != nil {
+			log.Error("Get http router path err", err)
+		}
+		name := route.GetName() //if not set name attribute,GetName return ""
+		if name != "" && err == nil {
+			httpRouterPage.WriteString("<tr><td><a href='" + pathTemplate + "'>" + name + "</a><td></tr>")
+		}
+		return nil
+	})
+	if err != nil {
+		fmt.Println(err)
+	}
+	httpRouterPage.WriteString("<tr><td><a href='/debug/pprof/'>Debug</a><td></tr>")
+	httpRouterPage.WriteString("</table></body></html>")
+	router.HandleFunc("/", func(responseWriter http.ResponseWriter, request *http.Request) {
+		_, err = responseWriter.Write([]byte(httpRouterPage.String()))
+		if err != nil {
+			log.Error("Http index page error", err)
```

Comment:
Do we need to add a blank space after "page error"?

Reply:
OK